### PR TITLE
Interrupt icon for single cell execution, fix nb toolbar icons

### PR DESCRIPTION
**Pull request by DonJayamanne**
For #4632

* Ability to interrupt when running a single cell
* Change order of icons
* Disable `Run Cell Above` if focus is set to first cell
* Change restart to black&wite (remove green color)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -798,32 +800,38 @@
                 {
                     "command": "jupyter.notebookeditor.runallcellsabove",
                     "title": "%DataScience.runAbove%",
-                    "group": "navigation",
-                    "when": "notebookViewType == 'jupyter-notebook' && jupyter.isnotebooktrusted && jupyter.havenativecells"
```
Comment (DonJayamanne) [inline package.json:802]: These icons never appeared.

File: package.json [hunk 2]
```diff
@@ -594,7 +594,9 @@
             {
                 "command": "jupyter.notebookeditor.interruptkernel",
                 "title": "%jupyter.command.jupyter.interruptkernel.title%",
-                "category": "Jupyter"
+                "category": "Jupyter",
+                "icon": "$(primitive-square)",
```
Comment (DonJayamanne) [inline package.json:598]: Use square (similar to VSCodes stop icon)

File: resources/light/restart-kernel.svg [hunk 3]
```diff
@@ -1,3 +1,3 @@
 <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path style="fill: #388A34 !important; fill-rule: evenodd !important; clip-rule: evenodd !important" d="M12.75 8C12.75 10.4853 10.7353 12.5 8.24999 12.5C6.41795 12.5 4.84162 11.4052 4.13953 9.83416L2.74882 10.399C3.67446 12.5186 5.78923 14 8.24999 14C11.5637 14 14.25 11.3137 14.25 8C14.25 4.68629 11.5637 2 8.24999 2C6.3169 2 4.59732 2.91418 3.5 4.3338V2.5H2V6.5L2.75 7.25H6.25V5.75H4.35201C5.13008 4.40495 6.58436 3.5 8.24999 3.5C10.7353 3.5 12.75 5.51472 12.75 8Z"/>
+<path style="fill: #424242 !important; fill-rule: evenodd !important; clip-rule: evenodd !important" d="M12.75 8C12.75 10.4853 10.7353 12.5 8.24999 12.5C6.41795 12.5 4.84162 11.4052 4.13953 9.83416L2.74882 10.399C3.67446 12.5186 5.78923 14 8.24999 14C11.5637 14 14.25 11.3137 14.25 8C14.25 4.68629 11.5637 2 8.24999 2C6.3169 2 4.59732 2.91418 3.5 4.3338V2.5H2V6.5L2.75 7.25H6.25V5.75H4.35201C5.13008 4.40495 6.58436 3.5 8.24999 3.5C10.7353 3.5 12.75 5.51472 12.75 8Z"/>
```
Comment (DonJayamanne) [inline resources/light/restart-kernel.svg:2]: No longer green

**Review by DavidKutu — APPROVED** [review]

File: src/client/common/application/notebook.ts [hunk 4]
```diff
@@ -32,6 +33,11 @@ export class VSCodeNotebook implements IVSCodeNotebook {
                   kernel: NotebookKernel | undefined;
               }>().event;
     }
+    public get onDidChangeNotebookEditorSelection(): Event<NotebookEditorSelectionChangeEvent> {
+        return this.canUseNotebookApi
+            ? this.window.onDidChangeNotebookEditorSelection
+            : new EventEmitter<NotebookEditorSelectionChangeEvent>().event;
```
Comment (joyceerhl) [inline src/client/common/application/notebook.ts:39]: Do we need to make sure we add these new EventEmitters to the disposable registry?

Comment (DonJayamanne) [reply src/client/common/application/notebook.ts]: Fixed.

**Review by IanMatthewHuff — APPROVED** [review]

**Pull request merged**